### ISSUE TITLE
[BUG] Tags and TagCategories screens jitter uncontrollably at certain display scales

### ISSUE THREAD

**Issue by LazyFace32** [FlashpointProject/launcher#355]
When you go to either the Tags or TagCategories (Categories) screen in the Flashpoint launcher and your windows display settings are set to a certain scale the two screens will start to jitter uncontrollably. The jittering is most noticeable on the Tags screen but can also be seen on the Categories screen most notably the "Description" title. It also seems to jitter most visibly on the right hand side of the Tags/Categories screen.

The issue only seems to happen at a certain display scaling, specifically at a Windows display scale of 175% or above. In a way it's also dependent on the resolution but only in the sense of what display scale the given resolution is able to support. Below is a list of the given resolutions and what scale the bug occurs at:

Problematic Resolutions:
2560x1440 at 175% scaling or above
1920x1200 at 175% scaling or above
1920x1080 at 175% scaling or above
1600x1200 at 175% scaling or above

Interestingly enough even though 1680x1050 supports a display scale of 175% it does not seem to make the bug occur. at least not enough to make it noticeable. The rest of the avilable resolutions (at least on Windows) does not seem to support a scaling of 175% so I was not able to reproduce the bug on those resolutions.

**To Reproduce**
1. (On Windows 10) Go to Windows System Settings > System > Display
2. Change the resolution to one of the Problematic Resolutions listed above
3. Change the Scale & Layout to at least 175% or above
4. Go to the Flashpoint launcher and go to the Tags or Categories tab to see the bug in action

**Expected behavior**
The expected behavior would of course be for the Tags and Categories screens not to jitter on these windows resolutions and scales.

**Screenshots**
Gyazo gif showing the bug in action:
https://gyazo.com/addd7d06a97e89ba2a0feec002317cff

**Desktop:**
 - OS: Windows 10
 - Flashpoint Version 10.1.0.7 Infinity - "Absence II"

**Comment by colin969** [comment]
I still can't get this reproduceable even with the exact same settings, odd

**Comment by LazyDkk** [comment]
Hmm I can try to provide some more information about my setup but I'm not sure if it'll be of any help. I'm running an Asus ROG Zephyrus G14 2021 gaming laptop.
Specs:
Screen: 1440p, 120Hz
Integrated GPU: ADM Radeon (TM) Graphics
Dedicated GPU: RTX 3060 Laptop GPU
CPU: Ryzen 9 5900HS Mobile

I'm usually running at a 1440p resolution with a display scale of 200% and a refresh rate at 60Hz (usually on battery, 120Hz when plugged in with the charger). After doing some testing I noticed a few things.

When the Flashpoint launcher uses the integrated gpu (AMD) it usually idles around 0-1% gpu usage but when ever I'm on the Tags or Categories tab in the launcher the usage increases to 43% on the Tags tab and 36% on the Categories tab. when the launcher is using the dedicated gpu (RTX 3060) the idle is also around 0-1%, sometimes lower. when I'm on the Tags tab the gpu usage increases to 38% and on the Categories tab it used around 29%. 

This drastic increase in gpu usage does not happen anywhere else in the launcher so perhaps there is a correlation? Additionally I noticed that a refresh rate of 120Hz made the jittering less obvious and that a refresh rate of 60Hz made it a lot more noticeable.

If there is any further information that I can provide, just ask.
